autoload/neomake/makers/ft/rust.vim

```diff
@@ -223,8 +231,9 @@ function! neomake#makers#ft#rust#CargoProcessOutput(context) abort
     return errors
 endfunction
 
-function! neomake#makers#ft#rust#FillErrorFromSpan(error, span) abort
-    let a:error.filename = a:span.file_name
+function! neomake#makers#ft#rust#FillErrorFromSpan(error, span, cwd) abort
+    let slash = neomake#utils#Slash()
```

Comment:
Could become `let s:slash = neomake#utils#Slash()` at the top of the script.

Reply:
fixed